sparta/CMakeLists.txt

```diff
@@ -136,7 +140,7 @@ if (CCACHE_PROGRAM)
 endif ()
 
 # -Wpedantic
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OPT_FLAGS} \
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OPT_FLAGS} ${LTO_FLAGS} \
```

Comment:
might wanna use add_compile_options() instead, it's the 'cmake way' to do this

Reply:
Done.  Thanks!